avro2py/codegen.py

```diff
@@ -338,9 +353,24 @@ def rewriter(self, module_namespace: str, module_imports: List[Union[ast.Import,
 class RewriteCrossReferenceStrings(ModuleAwareNodeTransformer):
     """Used to rewrite deferred type annotations within an AnnAssign node."""
 
+    def __init__(self, namespaces: Dict[str, Tuple[ast.AST, List[Union[ast.Import, ast.ImportFrom]]]]):
+        super(RewriteCrossReferenceStrings, self).__init__(namespaces)
+
+        namespace_parts = [namespace.split(".") for namespace in namespaces]
+        namespaces_with_children = [
+            ".".join(potential_parent)
+            for potential_parent, potential_child in itertools.permutations(namespace_parts, 2)
```

Comment:
This being n^2 instead of doing something like creating a tree or sorting and then comparing elements pairwise feels inelegant, but I guess it doesn't really matter.

Reply:
I think it seems a little inefficient but relatively straightforward.